lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,11 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.nil? or comments.last.nil?
```

Comment:
I think the function should return the same type (a string).  I believe this had caused issues before calling methods on `nil`